src/databricks/labs/remorph/reconcile/utils.py

```diff
@@ -0,0 +1,2 @@
+def alias_column_str(alias: str, columns: list[str]) -> list[str]:
```

Comment:
move this back to compare.py only I don't think this will warrant a new file of its own.